Source/Scene/PostProcessStageCollection.js

```diff
@@ -0,0 +1,675 @@
+define([
+        '../Core/Check',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Core/PixelFormat',
+        '../Renderer/PixelDatatype',
+        '../Renderer/Sampler',
+        '../Renderer/Texture',
+        '../Renderer/TextureMagnificationFilter',
+        '../Renderer/TextureMinificationFilter',
+        '../Renderer/TextureWrap',
+        '../Shaders/PostProcessStages/PassThrough',
+        './PostProcessStageLibrary',
+        './PostProcessStageTextureCache'
+    ], function(
+        Check,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        DeveloperError,
+        PixelFormat,
+        PixelDatatype,
+        Sampler,
+        Texture,
+        TextureMagnificationFilter,
+        TextureMinificationFilter,
+        TextureWrap,
+        PassThrough,
+        PostProcessStageLibrary,
+        PostProcessStageTextureCache) {
+    'use strict';
+
+    var stackScratch = [];
+
+    /**
+     * A collection of {@link PostProcessStage}s and/or {@link PostProcessStageComposite}s.
+     * <p>
+     * The input texture for each post-process stage is the texture rendered to by the scene or the texture rendered
+     * to by the previous stage in the collection.
+     * </p>
+     * <p>
+     * If the ambient occlusion or bloom stages are enabled, they will execute before all other stages.
+     * </p>
+     * <p>
+     * If the FXAA stage is enabled, it will execute after all other stages.
+     * </p>
+     *
+     * @alias PostProcessStageCollection
+     * @constructor
+     */
+    function PostProcessStageCollection() {
+        var fxaa = PostProcessStageLibrary.createFXAAStage();
+        var ao = PostProcessStageLibrary.createAmbientOcclusionStage();
+        var bloom = PostProcessStageLibrary.createBloomStage();
+
+        ao.enabled = false;
+        bloom.enabled = false;
+
+        var textureCache = new PostProcessStageTextureCache(this);
```

Comment:
We talked offline at one point, but remind me - there's no issues with multiple Cesium viewers / WebGL contexts since all caches are now per-scene, not static?

Reply:
Right there's no issue.